### ISSUE TITLE
kube-up fails under Google Cloud SDK 0.9.82

### ISSUE THREAD

**Issue by aronchick** [kubernetes/kubernetes#15760]
Don't know what caused it - just created a new configuration, and found

```
Waiting for group to become stable, current operations: creating: 4
Waiting for group to become stable, current operations: creating: 4
Waiting for group to become stable, current operations: creating: 4
Waiting for group to become stable, current operations: creating: 4
Waiting for group to become stable, current operations: creating: 1
Waiting for group to become stable, current operations: creating: 1
Waiting for group to become stable, current operations: creating: 1
Group is stable
You do not currently have this command group installed.  Using it
requires the installation of components: [app]


Your current Cloud SDK version is: 0.9.82
Installing components from version: 0.9.82


For the latest full release notes, please visit:
  https://dl.google.com/dl/cloudsdk/channels/rapid/RELEASE_NOTES

Do you want to continue (Y/n)?  Y

#============================================================#
#= Creating update staging area                             =#
#============================================================#
#= Installing: App Engine Command Line Interface (Preview)  =#
#============================================================#
#= Creating backup and activating new installation          =#
#============================================================#

Update done!

Restarting command:
  $ gcloud preview --project k8s-sample-project instance-groups --zone us-central1-b instances --group kubernetes-minion-group list

Usage: gcloud preview [optional flags] <group>
  group may be           app | datastore

Manage Preview CLI command groups.

optional flags:
  --account ACCOUNT      Google Cloud Platform user account to use for
                         invocation.
  --format FORMAT        The format for printing command output resources.
  --help                 Display detailed help.
  --log-http             Log all HTTP server requests and responses to stderr.
  --project PROJECT_ID   Google Cloud Platform project ID to use for this
                         invocation.
  --quiet, -q            Disable all interactive prompts.
  --trace-token TRACE_TOKEN
                         Token used to route traces of service requests for
                         investigation of issues.
  --user-output-enabled  Print user intended output to the console.
  --verbosity VERBOSITY  Override the default verbosity for this command.  This
                         must be a standard logging verbosity level: [debug,
                         info, warning, error, critical, none] (Default:
                         [warning]).
  -h                     Print a summary help and exit.

command groups:
  app                    *(BETA)* Manage your App Engine app.
  datastore              *(BETA)* Manage your Cloud Datastore.

ERROR: (gcloud.preview) Invalid choice: 'instance-groups'.
```

**Comment by aronchick** [comment]
@jlowdermilk

**Comment by aronchick** [comment]
Issue was addressed in 1.0.6.
